clang-tools-extra/clangd/refactor/Rename.cpp

```diff
@@ -1072,6 +1072,10 @@ llvm::Expected<RenameResult> rename(const RenameInputs &RInputs) {
   if (Reject)
     return makeError(*Reject);
 
+  static constexpr trace::Metric RenameTriggerCounter(
```

Comment:
can you do this at line 1065 instead? that way we can better track all the requests, and not just the succesfully served ones. (to guide us for improving the cases that are popular but don't work)